ios/Podfile.lock

```diff
@@ -1141,4 +1141,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: faae545c8a372673d0c73d49b0d4fcdcb8e778bf
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.2
```

Comment:
Skal denne egentlig byttes? Mulig det er lurt å installere samme versjon som resten her, med mindre andre skal oppdatere. 

```
gem uninstall cocoapods
gem install cocoapods -v 1.10.1

# kjør pod install igjen etter det er oppdatert
```

Mulig du må bruke `sudo` 🤔

Reply:
Litt usikker på hvordan det skjedde, tilbake på 1.10.1 nå, takk for heads-up!